PHPUnit 6.2.4 by Sebastian Bergmann and contributors.

EEEEEEEEEEEEEWEEEEEEEEEEEEEEEEE                                   31 / 31 (100%)

Time: 18.36 seconds, Memory: 66.00MB

There were 30 errors:

1) Magestore\Webpos\Api\Cart\CheckoutTest::testSaveCart
ReflectionException: Class Magestore\Webpos\Api\Cart\Constraint\Checkout does not exist

/var/www/vendor/magento/framework/Code/Reader/ClassReader.php:19
/var/www/vendor/magento/framework/ObjectManager/Definition/Runtime.php:44
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:48
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Cart/CheckoutTest.php:43
/var/www/vendor/phpunit/phpunit/phpunit:53

2) Magestore\Webpos\Api\Cart\CheckoutTest::testSaveQuoteData
ReflectionException: Class Magestore\Webpos\Api\Cart\Constraint\Checkout does not exist

/var/www/vendor/magento/framework/Code/Reader/ClassReader.php:19
/var/www/vendor/magento/framework/ObjectManager/Definition/Runtime.php:44
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:48
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Cart/CheckoutTest.php:43
/var/www/vendor/phpunit/phpunit/phpunit:53

3) Magestore\Webpos\Api\Cart\CheckoutTest::testSaveShippingMethod
ReflectionException: Class Magestore\Webpos\Api\Cart\Constraint\Checkout does not exist

/var/www/vendor/magento/framework/Code/Reader/ClassReader.php:19
/var/www/vendor/magento/framework/ObjectManager/Definition/Runtime.php:44
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:48
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Cart/CheckoutTest.php:43
/var/www/vendor/phpunit/phpunit/phpunit:53

4) Magestore\Webpos\Api\Cart\CheckoutTest::testPlaceOrder
ReflectionException: Class Magestore\Webpos\Api\Cart\Constraint\Checkout does not exist

/var/www/vendor/magento/framework/Code/Reader/ClassReader.php:19
/var/www/vendor/magento/framework/ObjectManager/Definition/Runtime.php:44
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:48
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Cart/CheckoutTest.php:43
/var/www/vendor/phpunit/phpunit/phpunit:53

5) Magestore\Webpos\Api\Cart\CheckoutTest::testApplyGiftcard
ReflectionException: Class Magestore\Webpos\Api\Cart\Constraint\Checkout does not exist

/var/www/vendor/magento/framework/Code/Reader/ClassReader.php:19
/var/www/vendor/magento/framework/ObjectManager/Definition/Runtime.php:44
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:48
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Cart/CheckoutTest.php:43
/var/www/vendor/phpunit/phpunit/phpunit:53

6) Magestore\Webpos\Api\Cart\CheckoutTest::testSpendPoint
ReflectionException: Class Magestore\Webpos\Api\Cart\Constraint\Checkout does not exist

/var/www/vendor/magento/framework/Code/Reader/ClassReader.php:19
/var/www/vendor/magento/framework/ObjectManager/Definition/Runtime.php:44
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:48
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Cart/CheckoutTest.php:43
/var/www/vendor/phpunit/phpunit/phpunit:53

7) Magestore\Webpos\Api\Catalog\CategoryRepositoryTest::testGetList
RuntimeException: Source class "\Magestore\Webpos\Api\Catalog\Constraint\CategoryInterface" for "Magestore\Webpos\Api\Catalog\Constraint\CategoryRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Catalog/CategoryRepositoryTest.php:29
/var/www/vendor/phpunit/phpunit/phpunit:53

8) Magestore\Webpos\Api\Catalog\ProductRepositoryTest::testGetList
RuntimeException: Source class "\Magestore\Webpos\Api\Catalog\Constraint\ProductInterface" for "Magestore\Webpos\Api\Catalog\Constraint\ProductRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Catalog/ProductRepositoryTest.php:30
/var/www/vendor/phpunit/phpunit/phpunit:53

9) Magestore\Webpos\Api\Catalog\ProductRepositoryTest::testGetOptions
RuntimeException: Source class "\Magestore\Webpos\Api\Catalog\Constraint\ProductInterface" for "Magestore\Webpos\Api\Catalog\Constraint\ProductRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Catalog/ProductRepositoryTest.php:30
/var/www/vendor/phpunit/phpunit/phpunit:53

10) Magestore\Webpos\Api\Catalog\SwatchRepositoryTest::testGetList
RuntimeException: Source class "\Magestore\Webpos\Api\Catalog\Constraint\SwatchInterface" for "Magestore\Webpos\Api\Catalog\Constraint\SwatchRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Catalog/SwatchRepositoryTest.php:29
/var/www/vendor/phpunit/phpunit/phpunit:53

11) Magestore\Webpos\Api\Checkout\CartTest::testSendEmail
Exception: <!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" >
<head>
    <title>There has been an error processing your request</title>
    <base href="http://magento2.travis/pub/errors/default/" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="robots" content="*"/>
    <link rel="stylesheet" href="css/styles.css" type="text/css" />
    <link rel="icon" href="images/favicon.ico" type="image/x-icon" />
    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" />
</head>
<body>
    <main class="page-main">
        
<h1>There has been an error processing your request</h1>

<div class="trace-container">
    <div class="trace">
        <pre>Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]</pre>
    </div>
</div>
    <p>Error log record number: 1274833190166</p>
    </main>
</body>
</html>


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Checkout/CartTest.php:38
/var/www/vendor/phpunit/phpunit/phpunit:53

12) Magestore\Webpos\Api\Checkout\TaxClassTest::testGetList
Exception: <!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" >
<head>
    <title>There has been an error processing your request</title>
    <base href="http://magento2.travis/pub/errors/default/" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="robots" content="*"/>
    <link rel="stylesheet" href="css/styles.css" type="text/css" />
    <link rel="icon" href="images/favicon.ico" type="image/x-icon" />
    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" />
</head>
<body>
    <main class="page-main">
        
<h1>There has been an error processing your request</h1>

<div class="trace-container">
    <div class="trace">
        <pre>Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]</pre>
    </div>
</div>
    <p>Error log record number: 1472363960727</p>
    </main>
</body>
</html>


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Checkout/TaxClassTest.php:40
/var/www/vendor/phpunit/phpunit/phpunit:53

13) Magestore\Webpos\Api\Config\ConfigRepositoryTest::testGetList
Exception: <!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" >
<head>
    <title>There has been an error processing your request</title>
    <base href="http://magento2.travis/pub/errors/default/" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="robots" content="*"/>
    <link rel="stylesheet" href="css/styles.css" type="text/css" />
    <link rel="icon" href="images/favicon.ico" type="image/x-icon" />
    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" />
</head>
<body>
    <main class="page-main">
        
<h1>There has been an error processing your request</h1>

<div class="trace-container">
    <div class="trace">
        <pre>Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]</pre>
    </div>
</div>
    <p>Error log record number: 1407026860655</p>
    </main>
</body>
</html>


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Config/ConfigRepositoryTest.php:33
/var/www/vendor/phpunit/phpunit/phpunit:53

14) Magestore\Webpos\Api\Customer\CustomerRepositoryTest::testGetList
RuntimeException: Source class "\Magestore\Webpos\Api\Customer\Constraint\CustomerInterface" for "Magestore\Webpos\Api\Customer\Constraint\CustomerRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Customer/CustomerRepositoryTest.php:29
/var/www/vendor/phpunit/phpunit/phpunit:53

15) Magestore\Webpos\Api\Customer\CustomerRepositoryTest::testSave
RuntimeException: Source class "\Magestore\Webpos\Api\Customer\Constraint\CustomerInterface" for "Magestore\Webpos\Api\Customer\Constraint\CustomerRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Customer/CustomerRepositoryTest.php:29
/var/www/vendor/phpunit/phpunit/phpunit:53

16) Magestore\Webpos\Api\Inventory\StockItemRepositoryTest::testGetStockItems
Exception: <!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" >
<head>
    <title>There has been an error processing your request</title>
    <base href="http://magento2.travis/pub/errors/default/" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="robots" content="*"/>
    <link rel="stylesheet" href="css/styles.css" type="text/css" />
    <link rel="icon" href="images/favicon.ico" type="image/x-icon" />
    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" />
</head>
<body>
    <main class="page-main">
        
<h1>There has been an error processing your request</h1>

<div class="trace-container">
    <div class="trace">
        <pre>Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]</pre>
    </div>
</div>
    <p>Error log record number: 1338650826128</p>
    </main>
</body>
</html>


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Inventory/StockItemRepositoryTest.php:50
/var/www/vendor/phpunit/phpunit/phpunit:53

17) Magestore\Webpos\Api\Payment\PaymentRepositoryTest::testGetList
RuntimeException: Source class "\Magestore\Webpos\Api\Payment\Constraint\PaymentInterface" for "Magestore\Webpos\Api\Payment\Constraint\PaymentRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Payment/PaymentRepositoryTest.php:29
/var/www/vendor/phpunit/phpunit/phpunit:53

18) Magestore\Webpos\Api\Pos\PosRepositoryTest::testGetList
RuntimeException: Source class "\Magestore\Webpos\Api\Pos\Constraint\PosInterface" for "Magestore\Webpos\Api\Pos\Constraint\PosRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Pos/PosRepositoryTest.php:36
/var/www/vendor/phpunit/phpunit/phpunit:53

19) Magestore\Webpos\Api\Pos\PosRepositoryTest::testAssignStaff
RuntimeException: Source class "\Magestore\Webpos\Api\Pos\Constraint\PosInterface" for "Magestore\Webpos\Api\Pos\Constraint\PosRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Pos/PosRepositoryTest.php:36
/var/www/vendor/phpunit/phpunit/phpunit:53

20) Magestore\Webpos\Api\Sales\CreditmemoRepositoryTest::testSaveCreditmemo
RuntimeException: Source class "\Magestore\Webpos\Api\Sales\Constraint\CreditmemoInterface" for "Magestore\Webpos\Api\Sales\Constraint\CreditmemoRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Sales/CreditmemoRepositoryTest.php:29
/var/www/vendor/phpunit/phpunit/phpunit:53

21) Magestore\Webpos\Api\Sales\InvoiceRepositoryTest::testUpdateQty
Exception: <!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" >
<head>
    <title>There has been an error processing your request</title>
    <base href="http://magento2.travis/pub/errors/default/" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="robots" content="*"/>
    <link rel="stylesheet" href="css/styles.css" type="text/css" />
    <link rel="icon" href="images/favicon.ico" type="image/x-icon" />
    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" />
</head>
<body>
    <main class="page-main">
        
<h1>There has been an error processing your request</h1>

<div class="trace-container">
    <div class="trace">
        <pre>Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]</pre>
    </div>
</div>
    <p>Error log record number: 1013484791992</p>
    </main>
</body>
</html>


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Sales/InvoiceRepositoryTest.php:66
/var/www/vendor/phpunit/phpunit/phpunit:53

22) Magestore\Webpos\Api\Sales\InvoiceRepositoryTest::testSaveInvoice
Exception: <!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" >
<head>
    <title>There has been an error processing your request</title>
    <base href="http://magento2.travis/pub/errors/default/" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="robots" content="*"/>
    <link rel="stylesheet" href="css/styles.css" type="text/css" />
    <link rel="icon" href="images/favicon.ico" type="image/x-icon" />
    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" />
</head>
<body>
    <main class="page-main">
        
<h1>There has been an error processing your request</h1>

<div class="trace-container">
    <div class="trace">
        <pre>Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]</pre>
    </div>
</div>
    <p>Error log record number: 1437416211978</p>
    </main>
</body>
</html>


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Sales/InvoiceRepositoryTest.php:80
/var/www/vendor/phpunit/phpunit/phpunit:53

23) Magestore\Webpos\Api\Sales\OrderRepositoryTest::testGetList
RuntimeException: Source class "\Magestore\Webpos\Api\Sales\Constraint\OrderInterface" for "Magestore\Webpos\Api\Sales\Constraint\OrderRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Sales/OrderRepositoryTest.php:29
/var/www/vendor/phpunit/phpunit/phpunit:53

24) Magestore\Webpos\Api\Sales\OrderRepositoryTest::testAddComment
RuntimeException: Source class "\Magestore\Webpos\Api\Sales\Constraint\OrderInterface" for "Magestore\Webpos\Api\Sales\Constraint\OrderRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Sales/OrderRepositoryTest.php:29
/var/www/vendor/phpunit/phpunit/phpunit:53

25) Magestore\Webpos\Api\Shift\CashTransactionRepositoryTest::testSave
RuntimeException: Source class "\Magestore\Webpos\Api\Shift\Constraint\CashTransactionInterface" for "Magestore\Webpos\Api\Shift\Constraint\CashTransactionRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Shift/CashTransactionRepositoryTest.php:35
/var/www/vendor/phpunit/phpunit/phpunit:53

26) Magestore\Webpos\Api\Shift\ShiftRepositoryTest::testGetList
RuntimeException: Source class "\Magestore\Webpos\Api\Shift\Constraint\ShiftInterface" for "Magestore\Webpos\Api\Shift\Constraint\ShiftRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Shift/ShiftRepositoryTest.php:35
/var/www/vendor/phpunit/phpunit/phpunit:53

27) Magestore\Webpos\Api\Shift\ShiftRepositoryTest::testSave
RuntimeException: Source class "\Magestore\Webpos\Api\Shift\Constraint\ShiftInterface" for "Magestore\Webpos\Api\Shift\Constraint\ShiftRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Shift/ShiftRepositoryTest.php:35
/var/www/vendor/phpunit/phpunit/phpunit:53

28) Magestore\Webpos\Api\Shift\ShiftRepositoryTest::testValidateSession
RuntimeException: Source class "\Magestore\Webpos\Api\Shift\Constraint\ShiftInterface" for "Magestore\Webpos\Api\Shift\Constraint\ShiftRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Shift/ShiftRepositoryTest.php:35
/var/www/vendor/phpunit/phpunit/phpunit:53

29) Magestore\Webpos\Api\Shift\ShiftRepositoryTest::testOpenSession
RuntimeException: Source class "\Magestore\Webpos\Api\Shift\Constraint\ShiftInterface" for "Magestore\Webpos\Api\Shift\Constraint\ShiftRepository" generation does not exist.

/var/www/vendor/magento/framework/Code/Generator.php:185
/var/www/vendor/magento/framework/Code/Generator.php:112
/var/www/vendor/magento/framework/Code/Generator/Autoloader.php:35
/var/www/vendor/magento/framework/ObjectManager/Relations/Runtime.php:38
/var/www/vendor/magento/framework/Interception/Config/Config.php:166
/var/www/vendor/magento/framework/Interception/Config/Config.php:189
/var/www/vendor/magento/framework/Interception/ObjectManager/Config/Developer.php:61
/var/www/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:47
/var/www/vendor/magento/framework/ObjectManager/ObjectManager.php:70
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Shift/ShiftRepositoryTest.php:35
/var/www/vendor/phpunit/phpunit/phpunit:53

30) Magestore\Webpos\Api\Staff\StaffManagementTest::testStaffLogin
Exception: <!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" >
<head>
    <title>There has been an error processing your request</title>
    <base href="http://magento2.travis/pub/errors/default/" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="robots" content="*"/>
    <link rel="stylesheet" href="css/styles.css" type="text/css" />
    <link rel="icon" href="images/favicon.ico" type="image/x-icon" />
    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" />
</head>
<body>
    <main class="page-main">
        
<h1>There has been an error processing your request</h1>

<div class="trace-container">
    <div class="trace">
        <pre>Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]</pre>
    </div>
</div>
    <p>Error log record number: 738461183633</p>
    </main>
</body>
</html>


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magestore/Webpos/Api/Staff/StaffManagementTest.php:33
/var/www/vendor/phpunit/phpunit/phpunit:53

--

There was 1 warning:

1) Warning
No tests found in class "Magestore\Webpos\Api\CurrentSessionId\CurrentSessionIdTest".

/var/www/vendor/phpunit/phpunit/phpunit:53

ERRORS!
Tests: 31, Assertions: 0, Errors: 30, Warnings: 1.

=== Memory Usage System Stats ===
Memory usage (OS):	95.01M (148.46% of 64.00M reported by PHP)
Estimated memory leak:	31.01M (32.64% of used memory)

